Formula/reminiscence.rb

```diff
@@ -39,6 +40,7 @@ def install
 
     ENV.prepend "CPPFLAGS", "-I#{libexec}/include"
     ENV.prepend "LDFLAGS", "-L#{libexec}/lib"
+    ENV.append "LDFLAGS", "-Wl,-rpath,#{libexec}/lib" unless OS.mac?
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-Wl,-rpath=#{libexec}/lib" unless OS.mac?
```
Stylistically I prefer to use `-rpath=` here rather than `-rpath,`